[{"id":583130434,"node_id":"R_kgDOIsHdQg","name":"CVE-2021-3426","full_name":"Live-Hack-CVE/CVE-2021-3426","private":false,"owner":{"login":"Live-Hack-CVE","id":121191732,"node_id":"O_kgDOBzk9NA","avatar_url":"https://avatars.githubusercontent.com/u/121191732?v=4","url":"https://api.github.com/users/Live-Hack-CVE","html_url":"https://github.com/Live-Hack-CVE","type":"Organization","site_admin":false},"html_url":"https://github.com/Live-Hack-CVE/CVE-2021-3426","description":"There's a flaw in Python 3's pydoc. A local or adjacent attacker who discovers or is able to convince another local or adjacent user to start a pydoc server could access the server and use it to disclose sensitive information belonging to the other user that they would not normally be able to access. The highest risk o CVE project by @Sn0wAlice","fork":false,"url":"https://api.github.com/repos/Live-Hack-CVE/CVE-2021-3426","created_at":"2022-12-28T21:25:36Z","updated_at":"2022-12-28T21:25:36Z","pushed_at":"2022-12-28T21:25:38Z","stargazers_count":0,"watchers_count":0,"forks_count":0,"archived":false,"disabled":false,"allow_forking":true,"is_template":false,"topics":[],"visibility":"public","forks":0,"watchers":0,"default_branch":"main","score":1}]